cookbooks/aws-parallelcluster-environment/resources/cfn_hup_configuration.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+#
+# Copyright:: 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+provides :cfn_hup_configuration
+unified_mode true
+default_action :configure
+
+action :configure do
+  cloudformation_url = "https://cloudformation.#{node['cluster']['region']}.#{node['cluster']['aws_domain']}"
+  instance_role_name = lambda {
+    # IMDS is not available on Docker
+    return "FAKE_INSTANCE_ROLE_NAME" if on_docker?
+    get_metadata_with_token(get_metadata_token, URI("http://169.254.169.254/latest/meta-data/iam/security-credentials"))
+  }.call
+
+  directory '/etc/cfn' do
+    owner 'root'
+    group 'root'
+    mode '0770'
```

Comment:
Here and below: why 0770 rather than 0700?
0700 is always preferable unless there is a use case where additional users belonging to the root group are expected to have same permission. In this case I think we should set 0700 because it's a folder supposed to be managed by us and we only use root user to do changes there

Reply:
I was keeping the old values for this directories. Anycase agree that we do not need to keep it 770